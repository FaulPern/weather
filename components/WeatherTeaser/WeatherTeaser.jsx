import WeatherIcons from '../WeatherIcons/WeatherIcons'

const WeatherTeaser = () => {
  return (
    <div className='teaser'>
      <WeatherIcons name='sun' />
      <span>13° / 19°</span>
    </div>
  )
}

export default WeatherTeaser
