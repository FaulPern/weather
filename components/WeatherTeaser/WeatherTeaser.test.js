import { mount } from 'enzyme'
import WeatherTeaser from './WeatherTeaser'

describe('WeatherTeaser component', () => {
  const mountComponent = mount(<WeatherTeaser />)
  it('should render', () => {
    expect(mountComponent).toMatchSnapshot()
  })
})
