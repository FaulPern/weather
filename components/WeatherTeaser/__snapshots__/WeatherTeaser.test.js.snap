// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherTeaser component should render 1`] = `
<WeatherTeaser>
  <div
    className="teaser"
  >
    <WeatherIcons
      name="sun"
    >
      <svg
        className=""
        height="10%"
        style={Object {}}
        viewBox="0 0 24.11 24.02"
        width="10%"
        xmlns="http://www.w3.org/2000/svg"
        xmlnsXlink="http://www.w3.org/1999/xlink"
      >
        <path
          d="M11.88.19a1,1,0,0,0-.88,1v2a1,1,0,0,0,2,0v-2a1,1,0,0,0-1-1ZM4,3.38a1,1,0,0,0-.81,1.16,1,1,0,0,0,.31.55L4.91,6.5a1,1,0,0,0,1.4.15,1,1,0,0,0,.15-1.41,1.06,1.06,0,0,0-.15-.15l-1.4-1.4a1,1,0,0,0-.72-.31H4Zm15.69,0a1,1,0,0,0-.6.31l-1.4,1.4a1,1,0,0,0-.15,1.41,1,1,0,0,0,1.4.15.54.54,0,0,0,.15-.15L20.5,5.09a1,1,0,0,0,0-1.41,1,1,0,0,0-.74-.3ZM12,5.19a7,7,0,1,0,7,7A7,7,0,0,0,12,5.19Zm0,2a5,5,0,1,1-5,5A5,5,0,0,1,12,7.19Zm-11.19,4a1,1,0,0,0-.9,1.09A1,1,0,0,0,1,13.19H3a1,1,0,0,0,0-2H.81Zm20,0a1,1,0,0,0-.9,1.09,1,1,0,0,0,1.09.91h2a1,1,0,0,0,0-2H20.81ZM5.47,17.59a.94.94,0,0,0-.56.32L3.5,19.28a1,1,0,1,0,1.41,1.44l1.4-1.41a1,1,0,0,0-.84-1.72Zm12.72,0a1,1,0,0,0-.81,1.17,1,1,0,0,0,.31.55l1.4,1.41a1,1,0,1,0,1.41-1.44l-1.41-1.37a1,1,0,0,0-.81-.32Zm-6.31,2.6a1,1,0,0,0-.88,1v2a1,1,0,0,0,2,0v-2a1,1,0,0,0-1-1Z"
          transform="translate(0.1 -0.18)"
        />
      </svg>
    </WeatherIcons>
    <span>
      13° / 19°
    </span>
  </div>
</WeatherTeaser>
`;
